andela/ah-backend-invictus
# #165273422 Enable users to receive real time notifications

**What does this PR do?**

- Create push notifications app
- use Django Pusher to send these notifications in real time via WebSockets
- Use Django signals to trigger and save notifications for authors when users do the following:
1. Like or dislike their article
2. Favourite their article
3. Comment on their article
4. Follow them

**How to test it out?**
As an authenticated user, either like, dislike, favourite or comment on an article.
While running your server, check out the page on localhost root directory and view the instant notification. 

A glimpse at real-time notification received when an article is liked. It informs the author of the particular article title

<img width="1438" alt="Screen Shot 2019-05-14 at 6 35 24 PM" src="https://user-images.githubusercontent.com/44635784/57711285-1b236100-7677-11e9-815b-1411e5503397.png">

[#165273422](https://www.pivotaltracker.com/story/show/165273422)
